test/integration/scheduler/framework_test.go

```diff
@@ -1477,3 +1535,27 @@ func TestPreemptWithPermitPlugin(t *testing.T) {
 	perPlugin.reset()
 	cleanupPods(cs, t, []*v1.Pod{waitingPod, preemptorPod})
 }
+
+func initTestContextForScorePlugin(t *testing.T, plugins *schedulerconfig.Plugins) (*testContext, *clientset.Clientset) {
+	// Create a plugin registry for testing. Register only a score plugin.
+	registry := framework.Registry{
+		scorePluginName:              NewScorePlugin,
+		scoreWithNormalizePluginName: NewScoreWithNormalizePlugin,
+	}
+
+	// Set empty plugin config for testing
+	emptyPluginConfig := []schedulerconfig.PluginConfig{}
+
+	// Create the master and the scheduler with the test plugin set.
+	context := initTestSchedulerWithOptions(t,
+		initTestMaster(t, "score-plugin", nil),
+		false, nil, registry, plugins, emptyPluginConfig, false, time.Second)
+
+	cs := context.clientSet
+	// Add multiple nodes, one of them will be scored much higher than the others.
```

Comment:
Is the part "one of them will be scored much higher than the others" correct?

Reply:
It's only relevant for the Score plugin. The Score plugin implementation should tell it anyways so I removed the comment.